packages-ui/roosterjs-react/lib/pasteOptions/plugin/createPasteOptionPlugin.ts

```diff
@@ -42,8 +42,6 @@ class PasteOptionPlugin implements ReactEditorPlugin {
                     this.clipboardData = clipboardData;
                     this.showPasteOptionPane();
                 }
-            } else {
-                this.pasteOptionRef.current?.dismiss();
```

Comment:
why remove?

Reply:
`this.pasteOptionRef.current` is always `null` according to TS due to a previous `else if`. `this.showPasteOptionPane` would change the ref, but that's in a block that won't get executed if execution reaches this `else`
https://github.com/microsoft/roosterjs/blob/e2706e52815ad7f664f2ebb4786b326f9b8c779e/packages-ui/roosterjs-react/lib/pasteOptions/plugin/createPasteOptionPlugin.ts#L34-L45